lib/tasks/whitehall_news_importer.rb

```diff
@@ -35,7 +37,39 @@ def create_document(translation, edition, document)
         locale: translation["locale"],
         title: translation["title"],
         publication_state: "changes_not_sent_to_draft",
+        summary: translation["summary"],
+        associations: associations(edition),
       )
     end
+
+    def associations(edition)
+      associations = {}
+      associations["primary_publishing_organisation"] = primary_publishing_organisation(edition)
+      associations["organisations"] = organisations(edition)
+      ASSOCIATIONS_WITH_ONLY_CONTENT_IDS.each do |type|
```

Comment:
I think it would be a bit clearer to just have the repeated lines here, which would actually decrease the total number of lines in the file. Can we do that for now?

Reply:
Fair enough, will do 👍